README.md

```diff
@@ -67,67 +65,81 @@ Configuration options should be configured in Hiera.  Defaults are defined in da
 
 ### Main system configuration
 
-##### `rsyslog::confdir`
+#### `rsyslog::confdir`
+
 Specifies the main directory where the module will place all configuration files (default: rsyslogd)
 
-##### `rsyslog::package_name`
+#### `rsyslog::package_name`
+
 The package to install (default: rsyslog)
 
-##### `rsyslog::package_version`
+#### `rsyslog::package_version`
+
 Package version/state to install, (default: installed)
 
-##### `rsyslog::feature_packages`
+#### `rsyslog::feature_packages`
+
 An array containing a list of extra packages (features) to install.
 
-##### `rsyslog::manage_package`
+#### `rsyslog::manage_package`
+
 Enable or disable managing the package (default: true)
 
-##### `rsyslog::manage_confdir`
+#### `rsyslog::manage_confdir`
+
 Enable or disable managing the configuration directory (confdir) (default: true)
 
-##### `rsyslog::purge_config_files`
+#### `rsyslog::purge_config_files`
+
 When `rsyslog::manage_confdir` is set to true, this option defines whether or not to purge unmanaged files within the configuration directory (default: true)
 
-##### `rsyslog::config_file`
+#### `rsyslog::config_file`
+
 Location of rsyslog main configuration file (default: /etc/rsyslog.conf)
 
-##### `rsyslog::override_default_config`
```

Comment:
There was some odd character or something on this line causing the markdown rendering to fail.